ui/server.py

```diff
@@ -18,11 +18,24 @@
 from starlette.responses import FileResponse, StreamingResponse
 from pydantic import BaseModel
 import logging
-
+# this is needed for development.
```

Comment:
this is really the only non "frontend" file that I had to update. Hope it all makes sense.